examples/basic/bars/nested_colormapped.py

```diff
@@ -7,6 +7,7 @@
     :keywords: bar, colormap, vbar
 
 '''
+from bokeh.palettes import MediumContrast3
 from bokeh.models import ColumnDataSource, FactorRange
 from bokeh.plotting import figure, show
```

Comment:
imports are out of order:
```suggestion
from bokeh.models import ColumnDataSource, FactorRange
from bokeh.palettes import MediumContrast3
from bokeh.plotting import figure, show
```

Reply:
Hello @bryevdv when i tried to rearrange it was giving an isort error..that was the only arrangement that it didnt..so i thought maybe that was the correct order

Reply:
@bryedv I have managed to correct it!..It was a mistake on my end...Kindly check the update